Error in <TSystem::ExpandFileName>: input: $HOME/.root.mimes, output: $HOME/.root.mimes
Warning in <TFile::Init>: file /eos/cms/store/group/dpg_mtd/comm_mtd/TB/MTDTB_H8_Sep2023/TOFHIR2/h8/reco//6195/70_e.root probably not closed, trying to recover
Info in <TFile::Recover>: /eos/cms/store/group/dpg_mtd/comm_mtd/TB/MTDTB_H8_Sep2023/TOFHIR2/h8/reco//6195/70_e.root, recovered key TTree:data at address 146204503
Warning in <TFile::Init>: successfully recovered 1 keys
Warning in <TFile::Init>: file /eos/cms/store/group/dpg_mtd/comm_mtd/TB/MTDTB_H8_Sep2023/TOFHIR2/h8/reco//6195/71_e.root probably not closed, trying to recover
Info in <TFile::Recover>: /eos/cms/store/group/dpg_mtd/comm_mtd/TB/MTDTB_H8_Sep2023/TOFHIR2/h8/reco//6195/71_e.root, recovered key TTree:data at address 30242810
Warning in <TFile::Init>: successfully recovered 1 keys
Warning in <TFile::Init>: file /eos/cms/store/group/dpg_mtd/comm_mtd/TB/MTDTB_H8_Sep2023/TOFHIR2/h8/reco//6195/8_e.root probably not closed, trying to recover
Warning in <TFile::Init>: no keys recovered, file has been made a Zombie

 *** Break *** segmentation violation



===========================================================
There was a crash.
This is the entire stack trace of all threads:
===========================================================
    gdb.printing.register_pretty_printer(gdb.current_objfile(),
    gdb.printing.register_pretty_printer(gdb.current_objfile(),
#0  0x00002b28fb19060c in waitpid () from /lib64/libc.so.6
#1  0x00002b28fb10df62 in do_system () from /lib64/libc.so.6
#2  0x00002b28f66893fc in TUnixSystem::StackTrace() () from /usr/lib64/root/libCore.so.6.24
#3  0x00002b28f668b31a in TUnixSystem::DispatchSignals(ESignals) () from /usr/lib64/root/libCore.so.6.24
#4  <signal handler called>
#5  0x000000000042ed25 in TTree* TDirectory::Get<TTree>(char const*) ()
#6  0x000000000042cf47 in TTree* TDirectoryFile::Get<TTree>(char const*) ()
#7  0x000000000042486f in main ()
===========================================================


The lines below might hint at the cause of the crash.
You may get help by asking at the ROOT forum https://root.cern.ch/forum
Only if you are really convinced it is a bug in ROOT then please submit a
report at https://root.cern.ch/bugs Please post the ENTIRE stack trace
from above as an attachment in addition to anything else
that might help us fixing this issue.
===========================================================
#5  0x000000000042ed25 in TTree* TDirectory::Get<TTree>(char const*) ()
#6  0x000000000042cf47 in TTree* TDirectoryFile::Get<TTree>(char const*) ()
#7  0x000000000042486f in main ()
===========================================================


Error in <TSystem::ExpandFileName>: input: $HOME/.root.mimes, output: $HOME/.root.mimes
Error in <TFile::TFile>: file /afs/cern.ch/user/s/spalluot/MTD/TB_CERN_Sep23/Lab5015Analysis/plots/moduleCharacterization_step1_HPK_nonIrr_LYSO813_Vov0.80_angle52_T5C.root does not exist

 *** Break *** segmentation violation



===========================================================
There was a crash.
This is the entire stack trace of all threads:
===========================================================
    gdb.printing.register_pretty_printer(gdb.current_objfile(),
    gdb.printing.register_pretty_printer(gdb.current_objfile(),
#0  0x00002b9d25c4860c in waitpid () from /lib64/libc.so.6
#1  0x00002b9d25bc5f62 in do_system () from /lib64/libc.so.6
#2  0x00002b9d211413fc in TUnixSystem::StackTrace() () from /usr/lib64/root/libCore.so.6.24
#3  0x00002b9d2114331a in TUnixSystem::DispatchSignals(ESignals) () from /usr/lib64/root/libCore.so.6.24
#4  <signal handler called>
#5  0x0000000000458144 in main ()
===========================================================


The lines below might hint at the cause of the crash.
You may get help by asking at the ROOT forum https://root.cern.ch/forum
Only if you are really convinced it is a bug in ROOT then please submit a
report at https://root.cern.ch/bugs Please post the ENTIRE stack trace
from above as an attachment in addition to anything else
that might help us fixing this issue.
===========================================================
#5  0x0000000000458144 in main ()
===========================================================


